plugins/modules/system/java_keystore.py

```diff
@@ -425,6 +425,7 @@ def create(self):
 
         import_keystore_cmd = [self.keytool_bin, "-importkeystore",
                                "-destkeystore", self.keystore_path,
+                               "-deststoretype", "jks",
```

Comment:
It would be nice to mention the minimum Java version in which this module will work. I see these options do not exist in older versions of `keytool`.

Reply:
Going further I think it might make sense to grab the java version from `keytool -J-version` and only append the option for Java > 1.8. And even then since the behavior has been variable based on Java version up to this point make the keystore type selectable and still default to PKCS12 for Java > 1.8.

Reply:
> It would be nice to mention the minimum Java version in which this module will work. I see these options do not exist in older versions of `keytool`.

I didn't even notice that, since this option exists in 1.7:
```
root@test6:~# keytool -J-version
openjdk version "1.7.0_261"
OpenJDK Runtime Environment (build 1.7.0_261-b)
OpenJDK 64-Bit Server VM (build 24.95-b-internal, mixed mode)
root@test6:~# keytool -importkeystore -help
keytool -importkeystore [OPTION]...

Imports one or all entries from another keystore

Options:

 -srckeystore <srckeystore>            source keystore name
 -destkeystore <destkeystore>          destination keystore name
 -srcstoretype <srcstoretype>          source keystore type
 -deststoretype <deststoretype>        destination keystore type
 -srcstorepass <arg>                   source keystore password
 -deststorepass <arg>                  destination keystore password
 -srcprotected                         source keystore password protected
 -srcprovidername <srcprovidername>    source keystore provider name
 -destprovidername <destprovidername>  destination keystore provider name
 -srcalias <srcalias>                  source alias
 -destalias <destalias>                destination alias
 -srckeypass <arg>                     source key password
 -destkeypass <arg>                    destination key password
 -noprompt                             do not prompt
 -providerclass <providerclass>        provider class name
 -providerarg <arg>                    provider argument
 -providerpath <pathlist>              provider classpath
 -v                                    verbose output
```